modules/rest-api/pages/rest-cluster-email-notifications.adoc

```diff
@@ -152,6 +154,15 @@ See xref:rest-api:rest-bucket-create.adoc[Creating and Editing Buckets], for inf
 Warns that the Index Service is, on a given node, occupying a percentage of available memory that is below an established threshold, which is the value of `lowIndexerResidentPerc`.
 The default value is `10`.
 
+* `maxDataDiskUsedPerc`. 
+The percentage of disk space used that will trigger an alert on the filesystem containing the data service, index service, or the `ns_log` or `audit_log`  storage paths.
+This alert warns you that the disk is becoming full.
+It occurs even if data disk usage limits are not enabled.
+The value must be an integer between `1` and `100`, which is the percentage of disk space used.
+It defaults to `90`.
```

Comment:
It actually defaults to 75%. Also, if the data disk limit is enabled, then it will ignore the configured threshold and use 10% less than the enforcement threshold.